docs/setup/install/auto-enroll.asciidoc

```diff
@@ -0,0 +1,15 @@
+[role="exclude"]
+
+If this is the first time you're starting {kib}, this command generates a 
+unique link to enroll your {kib} instance with {es}.
+
+. In your terminal, click the generated link to open {kib} in your browser.
+
+. In your browser, paste the enrollment token for {kib} and click the
+button to connect your {kib} instance with {es}.
+
+. Log in to {kib} as the `elastic` user with the password that was 
+generated when you started {es}.
+
+NOTE: If you need to reset the password for the `elastic` user or other
+built-in users, run the {ref}/reset-password.html[`elasticsearch-reset-password`] tool.
```

Comment:
This is great and super useful.

Maybe we could add a similar note for the enrollment token which can also be re-generated using a CLI in case it expires.

Reply:
I think it's more useful here than in the previous section as you'll only be interested in regenerating the token when an error occurs trying to enroll Kibana and that's what they will try at this stage.

Reply:
👍 I'll expand this note to include a link to the `elasticsearch-create-enrollment-token` tool and remove that mention from the previous section for generating an enrollment token.